### PR TITLE
feat: add chess slides + reveal render tool

### PR DESCRIPTION
**Pull request by thehowl**

### CODE REVIEW

**Comment by thehowl** [comment]
demo: https://thehowl.github.io/gno-workshops/presentations/2023-09-26--chess-the-gnolang-way--morgan/slides.html

File: .github/workflows/github-pages.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: github-pages
 
 on:
   push:
-    branches: [main]
+    branches: [dev/morgan/chess-slides-demo]
```
Comment (moul) [inline .github/workflows/github-pages.yml]:
```suggestion
    branches: [main]
```

File: .github/workflows/github-pages.yml [hunk 2]
```diff
@@ -16,9 +16,12 @@ jobs:
 
       - uses: actions/setup-go@v4
         with:
-          go-version: '1.20.x'
+          go-version: "1.20.x"
 
-      - run: go install golang.org/x/tools/cmd/present
+      - run: go install golang.org/x/tools/cmd/present ./cmd/reveal
```
Comment (moul) [inline .github/workflows/github-pages.yml]:
```suggestion
      - run: go install golang.org/x/tools/cmd/present
      - run: go install ./cmd/reveal
```

File: cmd/reveal/template.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	_ "embed"
+	"html/template"
+)
+
+// https://revealjs.com/config/
+// pointers everywhere so we don't accientally copy over stuff we don't need.
```
Comment (moul) [inline cmd/reveal/template.go]:
```suggestion
// pointers everywhere so we don't accidentally copy over stuff we don't need.
```

**Review by moul — APPROVED** [review]

**Pull request merged**